src/components/Layout.vue

```diff
@@ -126,6 +136,30 @@ export default class Layout extends Vue {
     window.location.reload(true)
   }
 
+  async getMetamaskNetwork() {
+    // @ts-ignore
+    await window.web3.version.getNetwork((err, networkId) => {
```

Comment:
Catch errors as this will throw if metamask is not present. Or call the ethereum/wallet/metamask#detect first